.travis.yml

```diff
@@ -1,27 +1,6 @@
-# Use container-based infrastructure
 sudo: false
-
 language: node_js
 node_js:
+  - "14"
   - "12"
   - "10"
-  - "8"
-
-matrix:
-  include:
-    # Test latest Node.js version also with a fresh install (no package-lock.json)
-    - node_js: 12
-      env: FRESH_INSTALL=true
-
-cache:
-  directories:
-    # Cache local npm cache files for faster install
-    - $HOME/.npm
-
-before_install:
-  # Remove package-lock.json to force a fresh install (updated dependencies)
-  - if [[ ${FRESH_INSTALL} == "true" ]]; then rm package-lock.json; fi
```

Comment:
Actually an interesting approach to test from consumer perspective

Reply:
Yeah, but I guess for us this isn't that important anymore as
- we regularly update the package-lock with the latest transitive dependencies
- we do such a test run in our release preflight job